dataland-internal-storage/src/main/kotlin/db/migration/V11__MigrateRefactoredDataPointClassesEuTaxoFinancials.kt

```diff
@@ -0,0 +1,132 @@
+package db.migration
+
+import db.migration.utils.DataTableEntity
+import db.migration.utils.MigrationHelper
+import db.migration.utils.getOrJavaNull
+import db.migration.utils.migrateCompanyAssociatedDataOfDatatype
+import org.flywaydb.core.api.migration.Context
+import org.json.JSONObject
+
+/**
+ * Performs the migration of the refactored data point classes for the eu taxonomy non fiancials framework
+ */
+class V11__MigrateRefactoredDataPointClassesEuTaxoFinancials {
+    /**
+     * Performs the migration of the refactored data point classes for eu taxonomy financials data
+     */
+    fun migrate(context: Context?) {
+        migrateCompanyAssociatedDataOfDatatype(
+            context,
+            "lksg",
+            this::migrateRefactoredDataPointClasses,
+        )
+    }
+    private val oldToNewFieldNamesForAssurance = mapOf(
+        "assurance" to "value",
+        "tagName" to "tagName",
```

Comment:
They are both the same. Why is this entry here?

Reply:
this was an attempt to fix the comparison in the migration test (the function "similar" didn't recognize similar entries where just the order is changed).
I just removed it.